camera_matrix: [893.509583 ,0.000000   ,632.748104 ,0.000000,
                0.000000   ,910.772888 ,393.341688 ,0.000000,
                0.000000   ,0.000000   ,1.000000   ,0.000000]

rlc: [0.999478   ,0.0301283  , 0.0116781,
          -0.0240397  ,0.9348230  ,-0.3542980,
          -0.0215914 ,0.3538320  , 0.9350600]

tlc: [ 0.7103788, 0.07362065, 0.07909130]


